tests/network/bindingplugin.go

```diff
@@ -230,11 +215,18 @@ var _ = Describe(SIG("network binding plugin", Serial, decorators.NetCustomBindi
 
 			serverVMI = libwait.WaitUntilVMIReady(serverVMI, console.LoginToAlpine)
 			clientVMI = libwait.WaitUntilVMIReady(clientVMI, console.LoginToAlpine)
-
 			Expect(libnet.AddIPAddress(serverVMI, guestIfaceName, serverCIDR)).To(Succeed())
 			Expect(libnet.AddIPAddress(clientVMI, guestIfaceName, clientCIDR)).To(Succeed())
+		})
+
+		It("can run a virtual machine with one primary managed-tap interface", func() {
+			Expect(serverVMI.Status.Interfaces).To(HaveLen(1))
+			Expect(serverVMI.Status.Interfaces[0].Name).To(Equal(networkName))
+		})
 
+		It("can establish communication between two VMs", func() {
 			Expect(libnet.PingFromVMConsole(clientVMI, serverIPAddr)).To(Succeed())
 		})
+
```

Comment:
the 2 It can be even combined to one test if desired, as checking iface exists + ping sounds pretty monolithic
and then no need `OncePerOrderedCleanup` etc

Reply:
`It` is usually used for tests that each test check a specific scenario / use case.
Here they can be declared as steps of the same use case, hence reduce mental complexity.